### PR TITLE
feat: add aspect ratio box with branding

### PR DESCRIPTION
**Pull request by saihaj**
### Summary of PR
An attempt to implement aspect ratio using css (as we know... need to use JS).  @bhajneet styled this to more branded look. Ratio box component (my next PR) is based of this so need this one to get merged first. 

### Reviewers
@Harjot1Singh 

Note: This is based from branch in #29 please approve that first.

### CODE REVIEW

File: src/App.css
Comment (harjot1singh) [inline src/App.css:17]: Are we still having this border, or was it meant for debugging only?

File: src/App.js
Comment (harjot1singh) [inline src/App.js]: No more than one newline gap. Same with L92-93

File: src/App.js
Comment (harjot1singh) [inline src/App.js]: If you are always passing `true` through, why pass it at all? Refactor to get rid of it.

We can also observe the pattern of `englishTranslation: TRANSLATION_ENGLISH` seems a little repetitive. A way of improving this would be to refactor to a default export of this object in the source file, and spreading it here.

**Review by harjot1singh — CHANGES_REQUESTED** [review]

File: src/App.css
Comment (saihaj) [reply src/App.css:17]: I have no clue 🤣. @bhajneet Do you know?

Comment (bhajneet) [reply src/App.css:17]: Yes we're keeping the border

File: src/App.js
Comment (saihaj) [reply src/App.js]: How do I make sure that spreading the variables will assign to the correct prop?

**Review by harjot1singh — APPROVED** [review]

**Comment by bhajneet** [comment]
I noticed the conflicting files, what is our standard operating procedure for this? Assignee or maintainer?

**Comment by saihaj** [comment]
Probably maintainer since:`Only those with write access to this repository can merge pull requests.`  
image
Also we can't merge this in since #29 needs to go first (this is a feature branch from that) and that will also bring some conflicts.

**Comment by bhajneet** [comment]
If this is on hold it should be a draft. Also I believe you can resolve merge conflicts and push the changes to this branch as well. But we can decide who does what later after #29 is merged as you said

**Comment by saihaj** [comment]
@Harjot1Singh since #29 let me know the next steps

**Comment by harjot1singh** [comment]
Too much merge conflict - did something go wrong? `SkipRender` is here - was it removed previously? Please resolve @saihaj

File: src/lib/options.js
Comment (harjot1singh) [inline src/lib/options.js]: This is still here
```suggestion
```

**Review by harjot1singh — CHANGES_REQUESTED** [review]

**Review by harjot1singh — APPROVED** [review]

**Pull request merged**
